src/App/Resources/AppResources.resx

```diff
@@ -2762,4 +2762,7 @@ Do you want to switch to this account?</value>
   <data name="LoggingInOn" xml:space="preserve">
     <value>Logging in on</value>
   </data>
+  <data name="BiometricsLoginExceeded" xml:space="preserve">
```

Comment:
⛏️ What do you think on appending the suffix `Description` to indicate that it's actually something larger than what's in the key? Btw I started using `DescriptionLong` as the key suffix for things that could be correct with just "Description" but in this case, I guess `Description` is just fine. `Explanation` could be another suffix.

Reply:
Sorry i was waiting for a final version and left it as a mock.